gc/base/MemoryManager.cpp

```diff
@@ -508,6 +519,16 @@ MM_MemoryManager::destroyVirtualMemory(MM_EnvironmentBase* env, MM_MemoryHandle*
 		}
 	}
 
+#if defined(OMR_ENV_DATA64) && !defined(OMR_GC_COMPRESSED_POINTERS)
```

Comment:
This is wrong place for this code. MM_MemoryManager::destroyVirtualMemory is called for any instance of Virtual Memory created, not just for object heap. I believe you can put it to MM_HeapVirtualMemory::tearDown()